package.json

```diff
@@ -78,7 +86,10 @@
     "compile-scripts": "tsc -p scripts-tsconfig.json",
     "fix": "gts fix",
     "pretest": "npm run compile",
-    "posttest": "npm run check"
+    "posttest": "npm run check",
+    "prepack": "npm run compile",
+    "pretest-install": "npm run compile",
+    "test-install": "mocha build/test/test-install.js"
```

Comment:
IIUC, `test-install` gets run either when we do `npm run test` or we do `npm run test-install`. Why do we need a special run script for it when test covers it?

Secondly, do you think it might be too expensive to include in the default test suite?

The client library team is actually testing the samples include in their repo. Have you looked at that? This smells a lot like a 'sample' that we should test to ensure it continues to work. Thoughts?

Reply:
I've removed the `test-install` since it is not needed.

I don't think these are too expensive to include in the default test suite.  Also, they are different from samples in that they test basic functionality, existence of types, etc. as apposed to displaying how to use the agent.